devops/scripts/set_docker_sock_permission.sh

```diff
@@ -4,5 +4,8 @@ set -e
 # By default the docker.sock file is not associated with docker group on codespaces or macOS
 # which causes a permission issue when docker is run without sudo.
 
-sudo chgrp docker /var/run/docker.sock
-sudo chmod g+rw /var/run/docker.sock
+if ! docker ps > /dev/null 2>&1; then
```

Comment:
Why is this important? What happens if we just update the permissions like it has been?

Reply:
Never mind, I see the customer issue